src/components/Calculator.jsx

```diff
@@ -0,0 +1,52 @@
+const Calculator = () => {
```

Comment:
- [ ] Based on the requirements, let's kindly make this a functional component. Please not ES6 functional component. **Hints**

`function Calculator () {}`